geomstats/learning/k_means.py

```diff
@@ -0,0 +1,87 @@
+import random
+
+import geomstats.backend as gs
+from geomstats.learning._template import TemplateTransformer
+
+
+class KMeans(TemplateTransformer):
+
+    def __init__(self, n_clusters, metric, init="random",
+                 tol=1e-2, verbose=0):
+        """ K-Means algorithm using Riemannian manifolds
+
+        Parameters
+        ----------
+        n_clusters : Number of clusters (k value of the k-means)
+
+        metric : The geomstats metric associate to the space used
+
+        init : How to init centroids at the beginning of the algorithm.
+               Only radom is implemented yet.
+
+        tol : convergence factor. If the difference of mean distance
+             between two step is lower than tol
+
+        verbose : if verbose > 0, information will be print during learning
+        Returns
```

Comment:
Cool, thanks for the doc!

Let's just add a line before Returns to separate the inputs and outputs?